crates/frontend/src/types.rs

```diff
@@ -271,7 +271,7 @@ impl From<ExperimentResponse> for Experiment {
 #[derive(Serialize, Deserialize, Clone, Debug)]
 pub struct Dimension {
     pub dimension: String,
-    pub priority: i32,
+    pub position: i32,
```

Comment:
should this have been u32 as well?

Reply:
@Datron its been handled in deserialization of Position value
if i change here to u32, in handler i have to do as i32 in few places , since in db to use u32 , i will have to write deserialization there